src/JSONParseErrorException.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/*
+    Copyright 2014 Rustici Software
```

Comment:
Copyright should match the year it was added, this should be either 2015 or 2016 at this point.